idaes/models/properties/modular_properties/transport_properties/no_method.py

```diff
@@ -0,0 +1,63 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+"Argh!" shouted the cyclops, "IDAES is throwing an error!"
```

Comment:
Whilst this is a very clever reference to Odysseus and Polyphemus, I am not sure it is something we really want in the code base (unfortunately).